sktime/utils/datetime.py

```diff
@@ -53,7 +53,8 @@ def _coerce_duration_to_int(duration, freq=None):
 def _get_freq(x):
     """Get unit for conversion of time deltas to integers"""
     if hasattr(x, "freqstr"):
-        return x.freqstr
+        #e.g. when freq = 'W-SUN'
+        return x.freqstr[0]
```

Comment:
So this change would simply ignore the `-SUN` information, is my understanding correct? If so, is that a good idea? In either case, we need some testing to make sure this works as expected.